### ISSUE TITLE
In 'Tools'

### ISSUE THREAD

**Issue by SieuDoTia** [MicrosoftDocs/typography-issues#1091]
TTOASM and TTODASM software only for TrueType font.

---
#### Document Details

⚠ *Do not edit this section. It is required for learn.microsoft.com ➟ GitHub issue linking.*

* ID: f6dafde5-8f87-72c7-63ef-a5e49b3b24d8
* Version Independent ID: ce1622bc-cd0c-6b55-e320-3344cf4ad463
* Content: [OpenType development (LEGACY INFORMATION) - Typography](https://learn.microsoft.com/en-us/typography/develop/otdevinfo)
* Content Source: [typographydocs/develop/OTDevInfo.md](https://github.com/MicrosoftDocs/typography/blob/live/typographydocs/develop/OTDevInfo.md)
* Product: **typography**
* GitHub Login: @nihar
* Microsoft Alias: **alib**

**Comment by alib-ms** [comment]
Two DOS utilities, TrueType Open Assembler (TTOAsm) and TrueType Open Disassembler (TTODasm), collaborate to assist in the creation, modification, and verification of TrueType Open (TTO) tables. TTOAsm accepts TrueType Open table data in text format and assembles that data into a binary TrueType Open table file. The output file is binary, following the specifications outlined in the Microsoft TrueType Open specification. A separate utility, AddTable, can be utilized to append the binary output file to a TrueType file.

Did I answer your question?

**Comment by SieuDoTia** [comment]
Yes, please add your comment in description about TTOASM and TTODASM.
